test/csi-addons/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+---
+resources:
+  - ${base_url}/crds.yaml
+  - ${base_url}/rbac.yaml
+  - ${base_url}/setup-controller.yaml
```

Comment:
instead of 3 you can use https://github.com/csi-addons/kubernetes-csi-addons/blob/main/deploy/controller/install-all-in-one.yaml which contains all 3 resources + webhooks for VRC and VR

Reply:
Do we need the web hooks? We try to have minimal setup that so you can run hub and 2 managed clusters on a laptop or in ci.

Reply:
it helps to catch any bugs if we have in webhooks as ramen is the only consumer of csiaddons Volume replication which has the right tests and setup's

Reply:
What is the cost in memory/cpu of adding the web hooks?

Reply:
I'll push this as is since I have other changes on top of this. We can improve this later if we want to add the web hooks.